dts/arm/atmel/sam4e.dtsi

```diff
@@ -303,6 +303,15 @@
 			clocks = <&pmc PMC_TYPE_PERIPHERAL 16>;
 			status = "disabled";
 		};
+
+		rtc: rtc@400e1860 {
+			compatible = "atmel,sam-rtc";
+			interrupts = <2 0>;
+			peripheral-id = <2>;
+			reg = <0x400e1860 0x100>;
```

Comment:
Respect following order:
- compatible
- regs
- interrupt
- clocks
- others

Update this for all

Reply:
updated